gui/src/app/login/login.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container-fluid pt-5 p-lg-5">
+  <h4 class="card-title">
```

Comment:
Since this page doesn't have so much content, I would center the title and the button.